src/core/js/views/questionView.js

```diff
@@ -320,6 +322,16 @@ define(function(require) {
 
         },
 
+        recordInteraction:function() {
+            var config = Adapt.config.get('_spoor');
```

Comment:
can we remove the spoor specific stuff from here? the framework should be agnostic., it should preferably do this check after the recordInteraction event